services/graph/pkg/service/v0/service.go

```diff
@@ -424,3 +427,19 @@ func parsePurgeHeader(h http.Header) bool {
 	}
 	return false
 }
+
+// ldap is using `=` signs as delimiter for key-values when doing requests
+// underlying go library is having problems when trying to parse names (keys) that contain `=`
```

Comment:
It would be nice if you'd mention the upstream issue here. I just found: https://github.com/go-ldap/ldap/issues/416 (even though it's talking about base64 encoded values, the issue is really about the `=` sign).